bin/p2-log-bridge/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"os"
+
+	"github.com/square/p2/pkg/logbridge"
+	"github.com/square/p2/pkg/logging"
+)
+
+func main() {
```

Comment:
I know it's a simple function, but it would be nice to at least support `--help` and give a short description of what the program is for.